pkgs/by-name/an/antimatter-dimensions/package.nix

```diff
@@ -0,0 +1,76 @@
+{ buildNpmPackage
+, electron
+, fetchFromGitHub
+, lib
+, makeDesktopItem
+, unstableGitUpdater
+, writeScriptBin
+, electronAppName ? "Antimatter Dimensions"
+}:
+
+let
+ dummyElectronApp = ./app.js;
+in
+buildNpmPackage rec {
```

Comment:
Maybe use `finalAttrs` instead of `rec`?

Reply:
tried `buildNpmPackage (self: {/*...*/})` however evaluation fails because buildNpmPackage doesn't support functions as argument.